test/assets/shipyard_onboard_service.yaml

```diff
@@ -54,7 +54,34 @@ spec:
                 deploymentstrategy: "direct"
             - name: "release"
 
-    - name: "production"
+    - name: "prod-a"
+      sequences:
+        - name: "delivery"
+          triggeredOn:
+            - event: "staging.delivery.finished"
+          tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "blue_green_service"
+            - name: "release"
+        - name: "rollback"
+          triggeredOn:
+            - event: "production.delivery.finished"
```

Comment:
I think this should be `prod-a.delivery.finished`

Reply:
thanks, has been adapted